app/views/wikis/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-   <div class="col-md-8">
+   <div class="col-md-10">
      <h2 class="text-center" style="margin: 20px;"><u>My wikis</u></h2>
      <% if policy_scope(@wikis).count > 0 %>
```

Comment:
@praveenyr the only change I'm going to request is for you to handle the policy_scope in the controller, not the view. I need clarification why you would wrap @wikis in a policy_scope when you did so in the controller already.